website/docs/r/slo.html.markdown

```diff
@@ -0,0 +1,153 @@
+    size            = "7d"
+    target          = 99
+    timezone        = "Asia/Kolkata"
+  }
+  indicator {
+    request_based_evaluation {
+      op         = "LessThanOrEqual"
+      query_type = "Logs"
+      threshold  = 1
+      queries {
+        query_group_type = "Threshold"
+        query_group {
+          row_id        = "A"
+          query         = <<QUERY
+          cluster=sedemostaging namespace=warp004*
+              | parse "Coffee preparation request time: * ms" as latency nodrop
+              |  if(isBlank(latency), "false", "true") as hasLatency
+              | where hasLatency = "true"
+              |  if(isBlank(latency), 0.0, latency) as latency
+              | latency/ 1000 as latency_sec
+QUERY
+          use_row_count = true
+          field = "latency_sec"
+        }
+      }
+    }
+  }
+}
+```
+
+## Argument reference
+
+The following arguments are supported:
+
+- `name` - (Required) The name of the SLO. The name must be alphanumeric.
+- `description` - (Optional) The description of the SLO.
+- `parent_id` - (Optional) The ID of the SLO Folder that contains this SLO. Defaults to the root folder.
+- `signal_type` - (Required) The type of SLO. Valid values are `Latency`, `Error`, `Throughput`, `Availability`
+  , `Other`. Defaults to `Latency`.
+- `service` - (Optional) The notifications the SLO will send when the respective trigger condition is met.
+- `application` - (Optional) Whether to group notifications for individual items that meet the trigger condition.
+  Defaults to true.
+- `compliance` - (Required) The compliance settings for the SLO.
+    - `compliance_type` - (Required) The type of compliance to use. Valid values are `Rolling` or `Calendar`.
+    - `target` - (Required) The target value to use, must be a number between 0 and 100.
+    - `timezone` - (Required) Time zone for the SLO compliance. Follow the format in the [IANA Time Zone Database][3].
+    - `size` - (Required) The size of the compliance period to use.
+      For `Calendar` compliance type the allowed values are `Week`, `Month`, `Quarter`.
+    - `start_from` - (Optional) Start of the calendar window. For week, it would be the day of the week (for e.g Sunday,
+      Monday etc). For month, it will always be the first day of the month. For quarter, it would be the first month of
+      the quarter (for e.g January, February etc.)
+    - `indicator` - (Required) The service level indicator on which SLO is to be defined. more details on the difference
+      b/w them can be found on
+      the [slo help page](https://help.sumologic.com/Beta/SLO_Reliability_Management/Access_and_Create_SLOs)
+        - [window_based_evaluation](#window_based_evaluation) - Evaluate SLI using successful/total windows.
+        - [request_based_evaluation](#request_based_evaluation) - Evaluate SLI based on occurrence's of successful
+          events / total events over entire compliance period.
+
+#### window_based_evaluation
```

Comment:
now that we have separate sections for window vs request, we can get rid of `events` wherever it is mentioned in this section and just use `window.`. Vice versa for request based section below.